components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/dto/APIInfoDTO.java

```diff
@@ -0,0 +1,105 @@
+package org.wso2.carbon.apimgt.rest.api.admin.dto;
+
+
+import io.swagger.annotations.*;
+import com.fasterxml.jackson.annotation.*;
+
+import javax.validation.constraints.NotNull;
+
+
+
+
+
+@ApiModel(description = "")
+public class APIInfoDTO  {
+  
+  
+  
+  private String name = null;
+  
+  
+  private String version = null;
+  
+  
+  private String provider = null;
+
+  private String lastUpdatedTime = null;
+
+  private String createdTime = null;
+
+  /**
+  * gets and sets the lastUpdatedTime for APIInfoDTO
+  **/
+  @JsonIgnore
+  public String getLastUpdatedTime(){
+    return lastUpdatedTime;
+  }
+  public void setLastUpdatedTime(String lastUpdatedTime){
+    this.lastUpdatedTime=lastUpdatedTime;
+  }
+
+  /**
+  * gets and sets the createdTime for a APIInfoDTO
+  **/
+
+  @JsonIgnore
+  public String getCreatedTime(){
```

Comment:
formatting issue

Reply:
Formatting issues needs to be fixed on the code generator side